masonite/view.py

```diff
@@ -130,11 +130,6 @@ def composer(self, composer_name, dictionary):
 
         return self
 
-    def extend(self):
-        """Extend class.
-        """
-
-        pass
 
```

Comment:
please remove this line too as Pep8 suggests 1 line space between methods in the class.